asistente_ladm_col/gui/controlled_measurement_dialog.py

```diff
@@ -114,31 +129,151 @@ def accept_dialog(self):
                 x_list.append(current_point.x())
                 y_list.append(current_point.y())
 
-            x_mean = statistics.mean(x_list)
-            y_mean = statistics.mean(y_list)
-            x_stdev = statistics.pstdev(x_list)
-            y_stdev = statistics.pstdev(y_list)
+            if x_list and y_list != []:
+                x_mean = statistics.mean(x_list)
+                y_mean = statistics.mean(y_list)
+                x_stdev = statistics.pstdev(x_list)
+                y_stdev = statistics.pstdev(y_list)
+            else:
+                continue
             geom = QgsGeometry.fromPointXY(QgsPointXY(x_mean, y_mean))
-
-            new_feature = QgsVectorLayerUtils.createFeature(layer, geom,
-                {
-                    0: group_id,
-                    1: len(x_list),
-                    2: x_mean,
-                    3: y_mean,
-                    4: x_stdev,
-                    5: y_stdev
-                }
-            )
+            fields_values.update({
+                layer.fields().indexOf("group_id"): group_id,
+                layer.fields().indexOf("count"): len(x_list),
+                layer.fields().indexOf("x_mean"): x_mean,
+                layer.fields().indexOf("y_mean"): y_mean,
+                layer.fields().indexOf("x_stdev"): x_stdev,
+                layer.fields().indexOf("y_stdev"): y_stdev
```

Comment:
```
fields = layer.fields()
fields_values.update({
    fields.indexOf("group_id"): group_id,
    fields.indexOf( ...
```

Reply:
solved in the commit: https://github.com/AgenciaImplementacion/Asistente-LADM_COL/commit/41a6511b624039fa0bb91702dbef2206a0e1bc83